lang/logtalk/Portfile

```diff
@@ -28,8 +28,8 @@ long_description    Logtalk is an object-oriented logic programming language tha
 homepage            http://logtalk.org/
 master_sites        ${homepage}files/
 
-checksums           rmd160  ad9e2f978229f89f378b38a5db7862565138234f \
-                    sha256  b65861b31da11bc416c9dd7bed1785821fce19b14cbe3d08d5a72f496ae45098
+checksums           rmd160  a3572cb0e54a64807584592dad0f57a4dafadfc0 \
+                    sha256  c175d0b70fcee71bc5228f39a936203ef8231fce272e9a7d189e2f1e58393425
```

Comment:
FYI, for your next update, we now also add a "size" line.

Reply:
Added. Thanks for the heads-up.

Reply:
Squish your changes into one git commit, though! :)